model/patch_lifecycle.go

```diff
@@ -173,16 +176,33 @@ func GetPatchedProject(ctx context.Context, p *patch.Patch, githubOauthToken str
 
 	// apply remote configuration patch if needed
 	if !(p.IsGithubPRPatch() || p.IsPRMergePatch()) && p.ConfigChanged(path) {
+		opts.ReadFileFrom = ReadFromPatchDiff
```

Comment:
right now, p.ConfigChanged() only checks if the main yaml was changed (in the case you tested in staging, evergreen.yml). What if _only_ an include file is changed? (ex.yml) 

I think what LoadProjectInto could do is in the loop `for _, path := range intermediateProject.Include`, check if `p.ConfigChanged(path)` (i.e. check it again for each of the included filenames), and if so change ReadFileFrom to ReadFromPatchDiff